src/entity/UserCompany.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Column,
+  Entity,
+  Index,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryColumn,
+} from 'typeorm';
+import { User } from './user';
+import { Company } from './Company';
+
+@Entity()
+export class UserCompany {
+  @Column({ default: () => 'now()' })
+  createdAt: Date;
+
+  @Column({ default: () => 'now()' })
+  updatedAt: Date;
```

Comment:
We can do `CreateDateColumn` i `UpdateDateColumn`

Reply:
What difference do they actually make under the hood?
(I will modify them)

Reply:
`UpdateDateColumn` should update when you do `save` on entity.